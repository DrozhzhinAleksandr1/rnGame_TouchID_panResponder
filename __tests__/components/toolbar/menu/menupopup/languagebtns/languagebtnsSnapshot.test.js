import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {LanguageBtns}  from '../../../../../../app/components/toolbar/menu/menupopup/languagebtns/languagebtns';

it('create snapshot LanguageBtns ', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<LanguageBtns />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
