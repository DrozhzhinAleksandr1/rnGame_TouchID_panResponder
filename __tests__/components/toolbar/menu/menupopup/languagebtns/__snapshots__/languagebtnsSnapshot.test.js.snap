// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot LanguageBtns  1`] = `
<Component
  style={
    Object {
      "flexDirection": "row",
      "height": 24,
      "marginLeft": 16,
      "marginTop": 32,
    }
  }
>
  <Component
    style={
      Object {
        "height": 24,
        "marginRight": 16,
        "width": 24,
      }
    }
  >
    <Icon_language />
  </Component>
  <LanguageBtn
    accessibilityLabel="BtnRuMenuID"
    styleBool={false}
    testID="BtnRuMenuID"
    text="Ru"
  />
  <LanguageBtn
    accessibilityLabel="BtnEnMenuID"
    styleBool={false}
    testID="BtnEnMenuID"
    text="En"
  />
  <LanguageBtn
    accessibilityLabel="BtnEsMenuID"
    styleBool={false}
    testID="BtnEsMenuID"
    text="Es"
  />
  <LanguageBtn
    accessibilityLabel="BtnArMenuID"
    styleBool={false}
    testID="BtnArMenuID"
    text="Ar"
  />
</Component>
`;
